api/lmes/v1alpha1/lmevaljob_types.go

```diff
@@ -478,6 +478,11 @@ type OCISpec struct {
 	// Path within the results to package as artifact
 	// +kubebuilder:validation:Pattern=`^[a-zA-Z0-9._/-]*$`
 	Path string `json:"path"`
+	// Subject for the OCI artifact
+	// +optional
+	// +kubebuilder:validation:Pattern=`^[a-zA-Z0-9._:/@-]*$`
+	// +kubebuilder:validation:MaxLength=255
+	Subject string `json:"subject,omitempty"`
```

Comment:
as this will be a OCI reference, I would validate that this reference starts with `registry` as prefix, wdyt?

ie, if
```
OCI_REGISTRY=quay.io
OCI_REPOSITORY=mmortari/my-artifact
```

then the
```
OCI_SUBJECT=quay.io/mmortari/my-model
```

starts with the same "server" OCI Registry (`quay.io`).

note: I don't find in the spec that the "org" must be the same, but it's likely a convention implemented by many registry, which I would not validate and eventually we response-code-fail if the permission setup from the server rejects it. I will followup on this point tho.